WooCommerce/Classes/ViewRelated/Feature Highlight/TooltipAnchor.swift

```diff
@@ -0,0 +1,86 @@
+import UIKit
+
+final class TooltipAnchor: UIControl {
```

Comment:
nit: it might be worth adding a comment on what this anchor means

Reply:
Removed this as we won't be needing this. https://github.com/woocommerce/woocommerce-ios/pull/10005/commits/da85d4e487f627f295a2e1e6f7fd116d47192776